pimd/pim_cmd.c

```diff
@@ -456,8 +455,6 @@ static void pim_show_membership(struct pim_instance *pim, struct vty *vty,
 			}
 		}
 	}
-
-	json_object_free(json);
```

Comment:
look at the non json loop this still needs to be here right?

Reply:
Really... reverting this.